Rubberduck.Core/UI/CodeExplorer/CodeExplorerWindow.cs

```diff
@@ -8,8 +9,8 @@ namespace Rubberduck.UI.CodeExplorer
     [ExcludeFromCodeCoverage]
     public partial class CodeExplorerWindow : UserControl, IDockableUserControl
     {
-        private const string ClassId = "C5318B59-172F-417C-88E3-B377CDA2D809";
-        string IDockableUserControl.ClassId { get { return ClassId; } }
+        private readonly string RandomGuid = Guid.NewGuid().ToString();
```

Comment:
:skull_and_crossbones:  makes me wonder what the purpose of `IDockableUserControl.GuidIdentifier` really is... do we really need this?